portfolio/src/main/webapp/index.html

```diff
@@ -1,37 +1,94 @@
+                            <li>PHYS051: Electromagnetism</li>
+                            <li>PHYS052: Quantum Physics</li>
+                        </ul>
+                    </div>
+                    <div class="text" style="flex: 0 0 33.4%; max-width: 33.4%;">
+                        <p class="text"><b>Computer Science</b></p>
+                        <ul style="margin-top: 0;">
+                            <li>CSCI005: Intro to Compuer Science (Python)</li>
+                            <li>CSCI060: Principles of Computer Science (Java &amp; Racket)</li>
+                            <li>CSCI070: Data Structures and Program Development (C++)</li>
+                        </ul>
+                    </div>
+                </div>
+                <p class="heading">Programming Languages and Tools</p>
+                <div class="flex-box">
+                    <div class="text" style="flex: 0 0 50%; max-width: 50%;">
+                        <p class="text"><b>Experienced</b></p>
+                        <ul style="margin-top: 0;">
+                            <li>Python</li>
+                            <li>Java</li>
+                            <li>C++</li>
+                            <li>Git</li>
+                        </ul>
+                    </div>
+                    <div class="text" style="flex: 0 0 50%; max-width: 50%;">
+                        <p class="text"><b>Familiar</b></p>
+                        <ul style="margin-top: 0;">
+                            <li>JavaScript</li>
+                            <li>CSS</li>
+                            <li>HTML</li>
+                            <li>MATLAB</li>
+                            <li>Pandas</li>
+                            <li>Scikit-Learn</li>
+                        </ul>
+                    </div>
+                </div>
+			</div>
+            <div class="front-page-column2">
+			    <div class="LI-profile-badge" data-version="v1" data-size="medium" data-locale="en_US" data-type="vertical" data-theme="dark" data-vanity="thomascintra"><a class="LI-simple-link" href='https://www.linkedin.com/in/thomascintra?trk=profile-badge'>Thomas Cintra</a></div>
+                <p class="text" style="margin-top: 10px"> Feel free to contact me at <a class="email" href="mailto:tcintra@g.hmc.edu">tcintra@g.hmc.edu</a>, or on any of the links below:</p>
+				<div class="media" style="margin-top: 10px">
+					<form action="https://github.com/Tcintra" target="_blank">
+						<button type="submit" class="media-btns"><i class="fab fa-github fa-3x"></i></button>
+					</form>
+					<form action="https://www.linkedin.com/in/thomascintra/" target="_blank">
+						<button type="submit" class="media-btns"><i class="fab fa-linkedin fa-3x"></i></button>
+					</form>
+					<form action="https://instagram.com/tncintra" target="_blank">
+						<button type="submit" class="media-btns"><i class="fab fa-instagram-square fa-3x"></i></button>
+					</form>
+				</div>
+            </div>
+		</div>
```

Comment:
Is it possible to apply some text formatting? It seems some tags are outlined (you could treat open, closing tags as curly braces in any programming language)

Reply:
By text formatting do you mean fixing indentation and such? I tried putting it all through an html beautifier but I wasn't sure what Google's style guidelines for html were.

Reply:
Yes, indentation. There is no strict guidelines, but overall it's better to keep indentation correct. I think there should be multiple options in the text editors to keep formatting correct. Also there are multiple online resources, where you put your code and it automatically format it: https://www.freeformatter.com/html-formatter.html

Reply:
Okay sounds good! I used http://beautifytools.com/html-beautifier.php but I don't think this website helped much, I'll use freeformatter.

Reply:
I pushed a new set of commits with some changes to home, new pages and hopefully fixed formatting issues. The command shift+ctrl+f (for me its shift+ctrl+i) wasn't doing much so I used freeformatter.